3rd_party/licenses/ipex-INFO.csv

```diff
@@ -0,0 +1,2 @@
+name,version,revision,url,license,copyright,sourceURL
+Intel® Extension for PyTorch,1.13.100+cpu,,https://github.com/intel/intel-extension-for-pytorch,Apache 2.0,(c) Intel Corporation,http://intel-optimized-pytorch.s3.cn-north-1.amazonaws.com.cn/libipex/cpu/libintel-ext-pt-cxx11-abi-1.13.100%2Bcpu.run
```

Comment:
The license needs to be `Apache-2.0` with a hyphen, so that it will match the identifier on SPDX: https://spdx.org/licenses/Apache-2.0.html

For Apache 2 dependencies we don't need the `sourceURL` link. That causes us to take a copy of the source code and make it available for download on our servers, and that's not necessary for Apache 2. So please leave `sourceURL` blank for this dependency.

I am also not sure about including the "®" symbol. It leaves us vulnerable to a character encoding bug far in the depths of the unified release process. I think it would be better to use "(R)", which won't cause a problem that's detected only on release day.

Finally, this file will get processed by shell commands, which like newline characters at the end of lines, so please make sure the second line ends with a newline character.

Reply:
Thank you for all this pointers!